DevEdu.API/DevEdu.API/Controllers/PaymentController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+using AutoMapper;
+using DevEdu.API.Models.InputModels;
+using DevEdu.Business.Servicies;
+using DevEdu.DAL.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DevEdu.API.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class PaymentController : Controller
+    {
+        private readonly IMapper _mapper;
+        private readonly IPaymentService _paymentService;
+
+        public PaymentController(IMapper mapper, IPaymentService paymentService)
+        {
+            _mapper = mapper;
+            _paymentService = paymentService;
+        }
+
+
+        //  api/payment/5
+        [HttpGet("{id}")]
+        public PaymentDto GetPayment(int id)
+        {
+            return _paymentService.GetPayment(id);
+        }
+
+        //  api/payment/by-user/1
+        [HttpGet("by-user/{userId}")]
+        public List<PaymentDto> GetPaymentByUserId(int userId)
+        {
+            return _paymentService.GetPaymentByUserId(userId);
+        }
+
+        //  api/payment
+        [HttpPost]
+        public int AddPayment([FromBody] PaymentInputModel model)
+        {
+            var dto = _mapper.Map<PaymentDto>(model);

```

Comment:
why don't you test your code?
![image](https://user-images.githubusercontent.com/18329725/125203688-2340b080-e282-11eb-83cc-032343cc7fad.png)